recipes/pydot/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pydot" %}
+{% set version = "1.2.3" %}
+{% set sha256 = "edb5d3f249f97fbd9c4bb16959e61bc32ecf40eee1a9f6d27abe8d01c0a73502" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
can this be `noarch: python`?

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages